cargo-rmc-tests/.gitignore

```diff
@@ -1,7 +0,0 @@
-empty-main/Cargo.lock
```

Comment:
This file appears as removed and then added somewhere else, instead of just moved. Because of that, the copyright check is failing.

Reply:
Hmm... I can do that. But then we will have to go through another PR to modify the contents.